src/assignment/components/blocks/AssignmentBlockEditSearch.tsx

```diff
@@ -7,11 +7,13 @@ import { WYSIWYG_OPTIONS_AUTHOR } from '../../../shared/constants';
 import { AssignmentBlockType, EditBlockProps } from '../../assignment.types';
 import { AssignmentBlockToggle } from '../AssignmentBlockToggle';
 
+import './AssignmentBlockEditSearch.scss';
+
 export const AssignmentBlockEditSearch: FC<EditBlockProps> = ({ block, setBlock }) => {
 	const [t] = useTranslation();
 
 	return (
-		<>
+		<div className="c-assignment-block-edit--search">
 			<TitleDescriptionForm
 				className="u-padding-l"
```

Comment:
```suggestion
		<>
			<TitleDescriptionForm
				className="u-padding-l c-assignment-block-edit--search"
```

Sidenote, het formulier is eerder een (B)E(M)lement `__` van `c-assignment-block-edit` dan een (BE)Mutator `--`